examples/step-50/step-50.cc

```diff
@@ -411,15 +431,15 @@ class LaplaceProblem
 
   DoFHandler<dim> dof_handler;
 
-  IndexSet                  locally_owned_set;
-  IndexSet                  locally_relevant_set;
+  IndexSet                  locally_owned_dofs;
+  IndexSet                  locally_relevant_dofs;
   AffineConstraints<double> constraints;
 
   MatrixType             system_matrix;
   MatrixFreeActiveMatrix mf_system_matrix;
   VectorType             solution;
   VectorType             right_hand_side;
-  Vector<double>         estimate_vector;
+  Vector<double>         error_estimator;
```

Comment:
I am not happy with the name "estimator" as this is a vector not a thing computing an estimate. How about ``error_estimate``?